### PR TITLE
Beam roblox patch 1

### PR DESCRIPTION
**Pull request by beam-roblox**
Add documentation for ColorGradingEffect

### CODE REVIEW

File: content/en-us/environment/post-processing-effects.md [hunk 1]
```diff
@@ -109,3 +110,14 @@ The `Class.SunRaysEffect` effect creates a halo of light with rays around the su
...
 <video src="../assets/lighting-and-effects/post-processing/Sun-Rays-Effect.mp4" controls width="800" alt="Video showing sun rays as the sun moves across the sky" ></video>
+
+## Color Grading
+
+The `Class.ColorGradingEffect` effect modifies how color values calculated by the renderer should be converted to the screen's color range, impacting the mood and appearance of your place.
+
+This effect's primary property is `Class.ColorGradingEffect.TonemapperPreset|TonemapperPreset`. The following values are available:
+
+- `Enum.TonemapperPreset.Default` &mdash; Sets the tone mapper to use the post‑2019 Roblox appearance which provides vivid colors and high contrasts.
+- `Enum.TonemapperPreset.Retro` &mdash; Sets the tone mapper to immitate the pre‑2019 Roblox appearance. Colors look less saturated and there's less contrast between them.
```
Comment (IgnisRBX) [inline content/en-us/environment/post-processing-effects.md:121]:
```suggestion
- `Enum.TonemapperPreset.Retro` &mdash; Sets the tone mapper to imitate the pre‑2019 Roblox appearance. Colors look less saturated and there's less contrast between them.
```

File: content/en-us/environment/post-processing-effects.md [hunk 2]
```diff
@@ -109,3 +110,23 @@ The `Class.SunRaysEffect` effect creates a halo of light with rays around the su
...
+- `Enum.TonemapperPreset.Retro` &mdash; Sets the tone mapper to imitate the pre‑2019 Roblox appearance. Colors look less saturated and there's less contrast between them.
+
+If you wish to recreate a full pre‑2019 Roblox look for your experience, experiment with `Enum.TonemapperPreset.Retro` and set the brightness of all lights to a maximum of `1.0`.
+
+<Tabs>
+<TabItem label="Default">
+<img src="../assets/lighting-and-effects/post-processing/ColorGrading-Default.jpg" width="800" height="450" alt="Experience without any effects applied" />
+</TabItem>
+<TabItem label="Retro">
+<img src="../assets/lighting-and-effects/post-processing/ColorGrading-Retro.jpg" width="800" height="450" alt="Experience with retro ColorGrading applied" />
```
[the remark this answers is not in the document]
Comment (beam-roblox) [reply content/en-us/environment/post-processing-effects.md:130]: Will be added in a separate PR

File: content/en-us/environment/post-processing-effects.md [hunk 3]
```diff
@@ -109,3 +110,23 @@ The `Class.SunRaysEffect` effect creates a halo of light with rays around the su
...
+This effect's primary property is `Class.ColorGradingEffect.TonemapperPreset|TonemapperPreset`. The following values are available:
+
+- `Enum.TonemapperPreset.Default` &mdash; Sets the tone mapper to use the post‑2019 Roblox appearance which provides vivid colors and high contrasts.
+- `Enum.TonemapperPreset.Retro` &mdash; Sets the tone mapper to imitate the pre‑2019 Roblox appearance. Colors look less saturated and there's less contrast between them.
+
+If you wish to recreate a full pre‑2019 Roblox look for your experience, experiment with `Enum.TonemapperPreset.Retro` and set the brightness of all lights to a maximum of `1.0`.
+
+<Tabs>
+<TabItem label="Default">
+<img src="../assets/lighting-and-effects/post-processing/ColorGrading-Default.jpg" width="800" height="450" alt="Experience without any effects applied" />
```
[the remark this answers is not in the document]
Comment (beam-roblox) [reply content/en-us/environment/post-processing-effects.md:127]: Will be added in a separate PR

File: content/en-us/reference/engine/classes/ColorGradingEffect.yaml [hunk 4]
```diff
@@ -2,8 +2,19 @@ name: ColorGradingEffect
...
+summary: |
+  Modifies how color values calculated by the renderer should be converted to the
+  screen's color range.
+description: |
+  The **ColorGradingEffect** effect modifies how color values calculated by the
+  renderer should be converted to the screen's color range, impacting the mood
+  and appearance of your place. 
+  
+  It is expected to be parented to `Class.Lighting`
+  and will be ignored if parented elsewhere. **ColorGradingEffect** instances cannot be combined. When multiple **ColorGradingEffect** instances are parented to `Class.Lighting`, only the most recently parented instance will be applied and the other instances will be ignored.
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/classes/ColorGradingEffect.yaml:14]:
```suggestion
  **ColorGradingEffect** is expected to be parented to `Class.Lighting` and will be ignored if parented
  elsewhere. Multiple instances cannot be combined and only the most recently parented instance to
  `Class.Lighting` will be applied.
```

File: content/en-us/reference/engine/enums/TonemapperPreset.yaml [hunk 5]
```diff
@@ -7,12 +7,12 @@ tags: []
 deprecation_message: ''
 items:
   - name: Default
-    summary: ''
+    summary: 'Sets the tone mapper to use the post‑2019 Roblox appearance which provides vivid colors and high contrasts.'
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/enums/TonemapperPreset.yaml:10]:
```suggestion
    summary: |
      Sets the tone mapper to use the post‑2019 Roblox appearance which provides vivid colors and high contrasts.
```

File: content/en-us/reference/engine/enums/TonemapperPreset.yaml [hunk 6]
```diff
@@ -7,12 +7,12 @@ tags: []
...
     value: 0
     tags: []
     deprecation_message: ''
   - name: Retro
-    summary: ''
+    summary: 'Sets the tone mapper to imitate the pre‑2019 Roblox appearance. Colors look less saturated and there is less contrast between them.'
```
Comment (IgnisRBX) [inline content/en-us/reference/engine/enums/TonemapperPreset.yaml:15]:
```suggestion
    summary: |
      Sets the tone mapper to imitate the pre‑2019 Roblox appearance. Colors look less saturated and there
      is less contrast between them.
```

**Review by IgnisRBX — APPROVED** [review]

**Pull request merged**